conf/app.routes

```diff
@@ -17,7 +17,12 @@ POST  /ated/:atedRefNo/property-details/has-value-change/:id        controllers.
 POST  /ated/:atedRefNo/property-details/acquisition/:id             controllers.PropertyDetailsValuesControllerImpl.saveDraftPropertyDetailsAcquisition(atedRefNo: String, id: String)
 POST  /ated/:atedRefNo/property-details/revalued/:id                controllers.PropertyDetailsValuesControllerImpl.saveDraftPropertyDetailsRevalued(atedRefNo: String, id: String)
 POST  /ated/:atedRefNo/property-details/owned-before/:id            controllers.PropertyDetailsValuesControllerImpl.saveDraftPropertyDetailsOwnedBefore(atedRefNo: String, id: String)
-POST  /ated/:atedRefNo/property-details/new-build/:id               controllers.PropertyDetailsValuesControllerImpl.saveDraftPropertyDetailsNewBuild(atedRefNo: String, id: String)
+## remove the old route rename for the new ones in the frontend
```

Comment:
Presumably this tag is no longer required

Reply:
Removed